docs/elastiflow-integration.md

```diff
@@ -0,0 +1,39 @@
+# Elastiflow Integration
+## Purpose
+Antrea supports sending IPFIX flow records as flow exporter. Elastiflow works
+as a data collector for flow records and flow-related information can be visualized
+in Kibana. This doc provides guidelines for deploying Elastiflow with support for
+Antrea-specific fields in a Kubenetes cluster.
+
+## About Elastiflow
+[Elastiflow](https://github.com/robcowart/elastiflow) is a network flow data
+collection and visualization tool based on [Elastic
+Stack](https://www.elastic.co/elastic-stack) (Elasticsearch, Logstash and
+Kibana). It supports Netflow v5/v9, sFlow and IPFIX flow types. We will use the
+IPFIX flow records with following fields:
```

Comment:
Add explicitly that we use ipfix protocol-- "Flow exporter feature in Antrea Agent uses IPFIX protocol to export flow records." 
Follow sentence can be:
IPFIX flow records contains following Antrea specific fields along with standard IANA fields.